db/seeds.rb

```diff
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+trail_1 = Trail.create(
+    name: "Mt Greylock", 
+    city: "Amherst", 
+    zip: "01234", 
+    route_type: "Loop", 
+    length: "3km",
+    elevation: "3km",
+    difficulty: "9",
+    description: "A description!"
+)
+trail_2 = Trail.create(
+    name: "Long, Long Trail", 
+    city: "West Stix", 
+    zip: "01235", 
+    route_type: "Winding Roads", 
+    length: "2km",
+    elevation: "100m",
+    difficulty: "3",
+    description: "You wil be lost but happy."
+)
```

Comment:
find_or_create_by is great to ensure if you run the seeds file multiple times you won't get duplicate data